tests/karma/unit/crmCaseTypeSpec.js

```diff
@@ -415,4 +442,40 @@ describe('crmCaseType', function() {
       });
     });
   });
+
+  /**
+   * Returns a sample api response for case type categories option values.
+   */
+  function getCaseTypeCategoriesSampleData() {
+    return {
+      values: [
+        {
+          "id": "1170",
+          "option_group_id": "153",
+          "label": "Workflow",
+          "value": "1",
+          "name": "WORKFLOW",
+          "filter": "0",
+          "is_default": "0",
```

Comment:
Should one of case type categories be `"is_default": "1"`?

Reply:
Good question. Test-wise I don't think it's necessary. The question is if we need to make an actual option value to be the default one. Will investigate.